provider/source/data/debug/duration/digital@1/fil.json

```diff
@@ -0,0 +1,6 @@
+{
+  "separator": ":",
+  "hms_hour_padding": 1,
+  "hm_hour_padding": 1,
+  "minute_padding": 1
+}
```

Comment:
As you can see, all the paddings are equal, so, we can just store one padding and returning an error in case they are different

Reply:
And as we discussed, if the leftmost padding is different , we can store two paddings in this case.